exec/csnex/Bytecode.cs

```diff
@@ -13,35 +13,36 @@ public class Bytecode
         public byte[] code;
 
         public List<FunctionInfo> functions;
+        public List<Cell> globals;
 
         public Bytecode()
         {
         }
 
-        public static UInt32 Get_VInt(byte[] pobj, ref uint i)
+        public static int Get_VInt(byte[] pobj, ref int i)
         {
-            uint r = 0;
+            int r = 0;
             while (i < pobj.Length) {
                 byte x = pobj[i];
                 i++;
                 if ((r & ~(uint.MaxValue >> 7)) != 0) {
```

Comment:
Should this also be changed to `int.MaxValue` to match the type?

Reply:
Fixed.